### PR TITLE
Fix notes

### PR DESCRIPTION
**Pull request by jnm2**
@333fred

### CODE REVIEW

File: meetings/2024/LDM-2024-07-17.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ can break collection expressions because we don't have a betterness tiebreaker a
 a different preferred span type than the rest of the language.
 
 Discussion in LDM showed a general reluctance to go with the narrow proposal, and we much preferred the version that applied better conversion from element recursively to solve this problem. That version
-gets us closer to unification with `params` collections and is more complete. While we do think this will be a problem in .NET 9, we're hopeful that we can use `OverloadResolutionPriorityAttribute` to solve
+gets us closer to unification with `params` collections and is more complete. While we do think this more complete version will ship later than .NET 9, we're hopeful that we can use `OverloadResolutionPriorityAttribute` to solve
```
Comment (333fred) [inline meetings/2024/LDM-2024-07-17.md:48]: The original wording is intentional here. `this` refers to overloading problem, not the solution.

Comment (jnm2) [reply meetings/2024/LDM-2024-07-17.md:48]: Okay, thanks. I'm still struggling to follow what it's saying. Why do we think that overloading will be a problem in .NET 9?

Comment (333fred) [reply meetings/2024/LDM-2024-07-17.md:48]: That's the motivating scenario we're as to why we've talked about this twice in as many weeks. Concat is a problem in NET 9.

Comment (jnm2) [reply meetings/2024/LDM-2024-07-17.md:48]: Sorry, I did get that, I just don't see it stated anywhere that better conversion from element is assumed not to be available in .NET 9, which would explain why Concat would then still be a problem in .NET 9. I can leave it though.

**Review by 333fred — APPROVED** [review]

**Pull request merged**